### ISSUE TITLE
[Feature request] `format-comment` distinct between doc comments and in-code comments

### ISSUE THREAD

**Issue by praxamarnix** [dart-code-checker/dart-code-metrics#1106]
**What do you want to discuss?**

I am using `format-comment` as a rule and would like to make a distinction between doc-style comments with three slashes (`///`) and simple in-code comments with two slashes (`//`).

I tried to add an ignore rule like so:

```yaml
- format-comment:
  exclude:
  - ^\/\/\s
```

But as the documentation describes, I believe that the slashes are removed from the input of the regex.

```dart
/// This is my very special class that requires a good sentence.
class Foo {
  void bar() {
    // first do something nice
    hello();
    // quick response
    bye();
  }
}
```

The simple comments inside the code can sometimes be a single word and I would be okay with ignoring those lines.
Only official documentation should be formatted.
Would it be possible to add an option for that?

**Comment by incendial** [comment]
@praxamarnix hi, sure, will add a configuration for this use-case. Please note that the `exclude` config entry is for excluding files, like so:
<img width="472" alt="Screenshot 2022-12-15 at 23 01 03" src="https://user-images.githubusercontent.com/21116742/207945085-5a31da5a-a0c0-4b65-97c3-e0472f57e486.png">

**Comment by incendial** [comment]
Available in 5.3.0 🚀
